### DISCUSSION TITLE
How to exclude some directories

### DISCUSSION THREAD

**Discussion by ghost** [rustic-rs/rustic#636] [Q&A] [answered]
In restic there is the option to put paths in a file like exclude.txt and then do `restic -r "/run/media/username/usb/data/backup" backup . --exclude-file="home/username/exclude.txt"`

I have also tried using a .gitignore file, putting it inside /home/username and also in /run/media/username/usb/data/backup. 

Could anyone help?

**Comment by aawsome** [comment, marked as answer]
You can use `--glob-file`, but you have to inverse the patterns, i.e. prepend a `!`, e.g.
```
!/my/file/pattern/*
```

**Reply by ghost** [reply]
I have added --glob-file="/home/user/exclude" I get the error: Error: No such file or directory (os error 2). The full command:
`sudo rustic -r /run/media/user/usb/data/bkp backup . --glob-file=home/son/exclude`

**Reply by aawsome** [reply]
Could it be that a `/` is missing in your filename? I think it should be `--glob-file=/home/son/exclude`.

**Reply by ghost** [reply]
Yes, now it appears to work! Thanks
